yarn-project/pxe/src/contract_function_simulator/pxe_oracle_interface.ts

```diff
@@ -637,7 +637,7 @@ export class PXEOracleInterface implements ExecutionDataProvider {
     noteHash: Fr,
     nullifier: Fr,
     txHash: TxHash,
-    recipient: AztecAddress,
+    scope: AztecAddress,
```

Comment:
```suggestion
    recipient: AztecAddress,
```

From the point of view of the contract, this is still the recipient. It is PXE that then chooses to store the note under the recipient's scope.